hrm-service/docker-database/docker-compose-persistent.yml

```diff
@@ -14,11 +14,20 @@
 
 services:
   db:
-    build: .
```

Comment:
Using build instead of just handling config in compose was surfacing strange build stage caching issues for me.